src/content/blog/ga_qas.md

```diff
@@ -0,0 +1,233 @@
+|       Mutation        |                                              Each gate on ansatz has a small probability of mutating to another gate (with the same number of qubits) in the pool                                               |
+
+_**Tab 1**. Detail of each operation in GA-QAS._
+
+The overall process can be viewed in _Fig. 2_.
+
+In GA, some hyper-parameters need to be considered and defined before you run GA-QAS:
+
+```py
+from qoop.evolution.environment import EEnvironmentMetadata
+env_metadata = EEnvironmentMetadata(
+        num_qubits, # As its name
+        depth, # Ansatz depth you want
+        num_circuit, # Number of ansatz per generation
+        num_generation, # Number of generation/iteration for GA
+        prob_mutate # Mutation probability, usually as small as 0.01 (1%)
+)
+```
+
+Then, you need to create an `EEnvironment` object, the important parameter is `fitness_func`, which is the function name that we declared above:
+
+```py
+from qoop.evolution.environment import EEnvironment
+env = EEnvironment(
+     metadata = env_metadata,
+     fitness_func = fitnessW,
+)
+```
+
+The object `EEnvironment` has other parameters such as `selection_func`, `crossover_func`, `mutate_func`, `threshold_func` which can be imported from below sub-module:
+
+```py
+from qoop.evolution import crossover, mutate, selection, threshold
+```
+
+or defined by yourself.
+
+|   Parrameter   |             Default             |                                                        Function type                                                         |
+| :------------: | :-----------------------------: | :--------------------------------------------------------------------------------------------------------------------------: |
+| selection_func |   selection.elitist_selection   |                 $f$: [qiskit.QuantumCircuit] $\times$ $\mathbb{R}^{N}$ $\rightarrow$ [qiskit.QuantumCircuit]                 |
+| crossover_func |  crossover.onepoint_crossover   | $f$: qiskit.QuantumCircuit $\times$ qiskit.QuantumCircuit $\rightarrow$ qiskit.QuantumCircuit $\times$ qiskit.QuantumCircuit |
+|  mutate_func   |     mutate.layerflip_mutate     |                                $f$: qiskit.QuantumCircuit $\rightarrow$ qiskit.QuantumCircuit                                |
+| threshold_func | threshold.compilation_threshold |                                             $f$: $\mathbb{R} \rightarrow$ {0,1}                                              |
+
+_**Table 2**: Parameter functions for GA-QAS._
+
+# Step 3. Run GA-QAS
+
+You can call the method `evol()` to start running GA-QAS.
+
+```py
+env.evol(
+   verbose = 1,
+   auto_save = True
+)
+```
+
+There are only two parameters: the first is verbose (1 means print % process, 0 means no print), and the second is the saving option.
```

Comment:
I'm not sure if this means something, but if it does, please explain.
```suggestion
There are only two parameters: the first is verbose (1 means print, 0 means no print), and the second is the saving option.
```

Reply:
It's like a verbose parameter from the deep learning framework, 1 means print the progress bar, 0 means no print anything (I added this explanation)

Reply:
Ohhhhh I see now. Print the percent **progress**. Right?